api/webscraper/utils/scraper_utils.py

```diff
@@ -11,14 +8,16 @@
 google_maps_api_key = os.environ.get('NEXT_PUBLIC_GOOGLE_MAPS_API_KEY')
 
 def check_status(status):
+    if status is None:
+        return None  # if status is missing, we will return None
     if status.lower() == 'cancelled':
         return 'Cancelled'
     elif status.lower() == 'operational' or status.lower() == 'completed':
         return 'Operational'
     elif status.lower() == 'under development':
         return 'Proposed'
     else:
-        return status # if status is None, we will return None
+        return status
```

Comment:
is the first if statement of return None and the else statement returning status redundant?

Reply:
I had this just in case the `status` wasn't one of the previous cases. For example, perhaps if there was some sort of misspelling in the data or if it was something weird like "Ongoing"? Do you want me to simply `None` if the status isn't one of our pre-determined cases, no matter what the actual status was?